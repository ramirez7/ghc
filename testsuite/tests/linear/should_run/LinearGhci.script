data T a = MkT a
:type +v MkT
:set -XLinearTypes
:type +v MkT
:set -XGADTs
data T a where MkT :: a ->. a -> T a
:info T
