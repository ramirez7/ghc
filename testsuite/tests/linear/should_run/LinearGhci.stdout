MkT :: a -> T a
MkT :: a -->.(n) T a
data T a where
  MkT :: a ->. a -> T a
  	-- Defined at <interactive>:6:1
